x/autopilot/keeper/liquidstake.go

```diff
@@ -72,12 +72,47 @@ func (k Keeper) RunLiquidStake(ctx sdk.Context, addr sdk.AccAddress, token sdk.C
 	}
 
 	msgServer := stakeibckeeper.NewMsgServerImpl(k.stakeibcKeeper)
-	_, err := msgServer.LiquidStake(
+	result, err := msgServer.LiquidStake(
 		sdk.WrapSDKContext(ctx),
 		msg,
 	)
 	if err != nil {
-		return errorsmod.Wrapf(sdkerrors.ErrInsufficientFunds, err.Error())
+		return errorsmod.Wrapf(err, err.Error())
 	}
-	return nil
+
+	if packetMetadata.IbcReceiver == "" {
+		return nil
+	}
+
+	hostZone, err := k.stakeibcKeeper.GetHostZoneFromHostDenom(ctx, token.Denom)
+	if err != nil {
+		return err
+	}
+
+	return k.IBCTransferStAsset(ctx, result.StToken, addr.String(), hostZone, packetMetadata)
+}
+
+func (k Keeper) IBCTransferStAsset(ctx sdk.Context, stAsset sdk.Coin, sender string, hostZone *stakeibctypes.HostZone, packetMetadata types.StakeibcPacketMetadata) error {
+	ibcTransferTimeoutNanos := k.stakeibcKeeper.GetParam(ctx, stakeibctypes.KeyIBCTransferTimeoutNanos)
+	timeoutTimestamp := uint64(ctx.BlockTime().UnixNano()) + ibcTransferTimeoutNanos
```

Comment:
I don't think the timeout matters that much, but from a UX perspective, I think my vote would be to use the default transfer timeout which is 10minutes (`transfertypes.DefaultRelativePacketTimeoutTimestamp`)

Reply:
We use 30 min elsewhere in the codebase, and given the UX with the fallback address is pretty painful, imo we should keep the higher timeout (it might help in the case that lots of blocks are full). although 10 vs. 30 min are both arbitrary and we could go higher

Reply:
The fallback address is only used for failures - timeouts are retried infinitely

Reply:
This was changed and is being discussed here https://github.com/Stride-Labs/stride/pull/1038#discussion_r1443768561